Gemfile.lock

```diff
@@ -229,6 +230,7 @@ GEM
     pathutil (0.16.2)
       forwardable-extended (~> 2.6)
     public_suffix (3.1.1)
+    racc (1.5.2)
```

Comment:
FWII this is not sketchy, it was racc was explicitly added is a dependency of nokogiri in [this](https://nokogiri.org/CHANGELOG.html#notes) release.

Reply:
Any idea about the other version updates?

Reply:
They're just minor bumps. I'd say we don't **need** to bump but some of the deps had security patches, nothing serious, but also nothing drastic.

Reply:
Ah wait, let me see if we should bump racc to the same versions.